src/ui/CustomStyleEditor.js

```diff
@@ -1118,7 +1122,7 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
                       <input
                         checked={this.state.styles.boldNumbering}
                         className="molsp-chkboldnumbering"
-                        disabled={this.state.styles.hasNumbering ? false : true}
+                        disabled={!(this.state.styles.hasNumbering) || (this.state.styles.styleLevel === 'None' || this.state.styles.styleLevel === undefined) ? true : false}
```

Comment:
Ambiguity on the '!' intent looks like a possible mistake. The '!' should either be inside the bubble, or the bubble should cover more than 1 expression.